Makefile

```diff
@@ -135,7 +135,7 @@ RGBU_VERSION ?= 2.34.0-2020.07.0
 ROCD_VERSION ?= 0.10.0-2020.04.6
 RQEMU_VERSION ?= 4.2.0-2020.04.0
 XC3SP_VERSION ?= 0.1.2-2020.04.0
-TDC_VERSION ?= 0.9.0-2020.04.1
+TDC_VERSION ?= 0.9.0-2020.06.0
```

Comment:
I would prefer that we kept the 2020.04... number to show that they are being released together